client/my-sites/checkout/composite-checkout/test/composite-checkout-variant-picker.tsx

```diff
@@ -149,11 +149,29 @@ describe( 'CompositeCheckout with a variant picker', () => {
 			const cartChanges = { products: [ getBusinessPlanForInterval( cartPlan ) ] };
 			render( <MyCheckout cartChanges={ cartChanges } /> );
 
-			const getVariantItemText = await screen.findByText(
+			const editOrderButton = await screen.findByLabelText( 'Edit your order' );
+			fireEvent.click( editOrderButton );
+
+			expect( screen.getByText( getVariantItemTextForInterval( expectedVariant ) ) ).toBeVisible();
+		}
+	);
+
+	it.each( [
```

Comment:
this new test ensures that a WordPress.com cart is not using the new Jetpack cart behavior of showing all variants.